src/main/java/io/jenkins/plugins/models/PluginReleases.java

```diff
@@ -0,0 +1,18 @@
+package io.jenkins.plugins.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.ArrayList;
+
+public class PluginReleases {
+  public ArrayList<PluginRelease> getReleases() {
+    return releases;
+  }
+
+  @JsonProperty("releases")
+  private final ArrayList<PluginRelease> releases;
```

Comment:
fields should be declared above methods, and method below constructor

normally you would use `List` instead of `ArrayList` as you could later change the type of list without having to worry about it being a breaking change, consumers just need to adhere to the list interface and not worry about the implementation